app/src/main/java/ch/sdp/vibester/user/UserScoreboardAdapter.kt

```diff
@@ -56,5 +57,6 @@ class UserScoreboardAdapter(playersInit: MutableList<User>, private val genre: S
             itemView.findViewById<TextView>(R.id.tv_score).text = setScore(player).toString()
             itemView.findViewById<ImageView>(R.id.iv_photo).loadImg(player.image)
         }
+
```

Comment:
Why extra empty space here, when there are no spaces between the two block closures below?